README.md

```diff
@@ -1,122 +1,241 @@
 # krill-sync
 
-A tool to synchronize an RRDP and/or Rsync server with a "hidden" remote [RFC 8182](https://tools.ietf.org/html/rfc8182) RRDP publication point. _**Note:** currently only tested with and makes assumptions specific to [Krill](https://nlnetlabs.nl/projects/rpki/krill/)._
+A tool to synchronize an RRDP and/or Rsync server with a "hidden" remote 
+[RFC 8182](https://tools.ietf.org/html/rfc8182) RRDP publication point.
 
-## TL;DR
+## Installation
+
+### Install using Pre-built Packages
+
+Assuming you have a machine running a recent Debian or Ubuntu distribution, you 
+can install krill-sync from our [software package repository](https://packages.nlnetlabs.nl).
+To use this repository, add the line below that corresponds to your operating system to 
+your `/etc/apt/sources.list` or `/etc/apt/sources.list.d/`
+
+```bash
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/debian/ stretch main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/debian/ buster main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/ubuntu/ xenial main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/ubuntu/ bionic main
+deb [arch=amd64] https://packages.nlnetlabs.nl/linux/ubuntu/ focal main
+```
+Then run the following commands.
+
+```bash
+sudo apt update && apt-get install -y gnupg2
+wget -qO- https://packages.nlnetlabs.nl/aptkey.asc | sudo apt-key add -
+sudo apt update
+```
+
+You can then install, enable and start Krill by running
+
+```bash
+sudo apt install krill-sync
+```
+
+
+### Build with Cargo
 
 For Ubuntu 20.04 with Rust 1.51.0:
 
 ```
 $ apt update && apt install -y build-essential curl libssl-dev openssl pkg-config
```

Comment:
Leave out the `$` to be consistent and allow easy copying.

Reply:
done